features/eolearn/features/feature_manipulation.py

```diff
@@ -252,38 +252,45 @@ class SpatialResizeTask(EOTask):
     def __init__(
         self,
         *,
-        resize_parameters: Tuple[ResizeParam, Tuple[float, float]],
+        resize_type: ResizeParam,
+        height_param: float,
+        width_param: float,
         features: FeaturesSpecification = ...,
         resize_method: ResizeMethod = ResizeMethod.LINEAR,
         resize_library: ResizeLib = ResizeLib.PIL,
     ):
         """
         :param features: Which features to resize. Supports new names for features.
-        :param resize_parameters: Instructions on how to perform the resizing process. For example use:
-            `['new_size', [500, 1000]]` to resize the data to size (500, 1000),
-            `['resolution', [10, 20]]` for changing resolution from 10 to 20,
-            `['scale_factors', [3, 3]]` to make the data three times larger.
+        :param resize_type: Type of resizing process to be performed. Options:
+            * `new_size` (combined with width_param, height_param). Resizes data to size (width_param, height_param)
+            * `resolution` (combined with width_param, height_param). Resizes the data to have
+               width_param, height_param resolution over width/height axis. Uses EOPatch bbox to compute.
+            * `scale_factor` (combined with width_param, height_param). Resizes the data by scaling the width
+               and height by a factor set by width_param and height_param.
```

Comment:
since you have `(combined with width_param, height_param)` maybe you can add that on the first line. Something along the lines of
```
:param resize_type: Determines type of resizing process and how `width_param` and `height_param` are used. Options:
```

Reply:
Thanks. Was struggling with wording this. The suggestion is much better than my version. Will fix.